packages/core/src/components/sections/Breadcrumb/DefaultComponents.ts

```diff
@@ -1,6 +1,7 @@
-import { Breadcrumb as UIBreadcrumb, Icon as UIIcon } from '@faststore/ui'
+import { Icon as UIIcon } from '@faststore/ui'
+import Breadcrumb from 'src/components/ui/Breadcrumb'
 
 export const BreadcrumbDefaultComponents = {
-  Breadcrumb: UIBreadcrumb,
+  __experimentalBreadcrumb: Breadcrumb,
```

Comment:
todo: After merging this PR we have to update the [doc](https://developers.vtex.com/docs/guides/faststore/building-sections-list-of-native-sections)

Instead of `Breadcrumb` should be listed `__experimentalBreadcrumb`.

<img width="620" alt="image" src="https://github.com/user-attachments/assets/cfb3d9f2-e283-4afb-b9ee-3f08d6580994" />